src/test/java/no/unit/alma/commons/HttpStatusExceptionTest.java

```diff
@@ -0,0 +1,35 @@
+package no.unit.alma.commons;
+
+import no.unit.alma.generated.error.WebServiceResult;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+class HttpStatusExceptionTest {
+
+
+    @Test
+    void testHttpStatusException() {
+        int status = 23;
+        String statusText = "sample status text";
+        String method = "sample method";
+        String url = "https://www.example.com";
+        String responseBody = "sample body";
+        String resultAsString = "sample result string";
+        WebServiceResult webServiceResult  = new WebServiceResult();
+        webServiceResult.setResult(resultAsString);
+
+        HttpStatusException httpStatusException = new HttpStatusException(status, statusText, method, url, responseBody, webServiceResult);
```

Comment:
Checkstyle sier at linja er for langt.

Reply:
hvorfor virker ikke min checkstyle ? :-(

Reply:
Hjelper kanskje å kjøre gradle clean build?

Reply:
![Bildschirmfoto 2020-03-27 um 06 36 01](https://user-images.githubusercontent.com/20472251/77725355-89ee7c00-6ff5-11ea-93d5-85d44dc86f43.png)

Reply:
manglet å velge kjøring av checkstyle på testene også i intellij (clean hjalp også :-)